### ISSUE TITLE
Changing Hyper-parameters to improve performance

### ISSUE THREAD

**Issue by setarekhosravi** [Kou-99/ZoomTrack#4]
Thank you for your great work, after many evaluations, I found that your model will work well on fast objects!
Now I have a problem, I've checked the performance of your model on my videos and I've evaluated it. I want to change the test hyperparameters to figure out how they affect performance during inference. (I don't want to train again, I just want to check the effect of test hyperparameters) 
but when I change the hyperparameters such as test.template.factor there is no effect on performance I think the parameters won't change. would you please guide me? 
Thank you!

**Comment by Kou-99** [comment]
Hi there! Thank you for your interest in our project. Regarding your inquiry, I haven't conducted experiments specifically altering the template factor, so I can't definitively say if adjusting it will impact tracking accuracy. However, modifying "test.template.factor" will indeed alter this factor. You can validate this by running the tracker in debug mode, for example: 
```python
python tracking/test.py ostrack zoom --dataset lasot --debug 5
```
Please ensure you have Visdom running before executing the tracker. Once set up, you can view the visualization results at localhost:8097. If you encounter any issues or have further questions, feel free to ask!

**Comment by setarekhosravi** [comment]
@Kou-99 
Thank you for your answer, I will try.
I have another question, I want to train your model on my custom dataset, is it possible? if it is possible how can I do it?
How to modify the dataset, paths, and ... to train your model?

**Comment by Kou-99** [comment]
The training datasets are located under lib/train/dataset. To create your customized training dataset, simply mimic the files within this directory. Once you've crafted your custom dataset, register it within the names2datasets function in lib/train/base_functions.py. Following this, integrate your custom dataset by appending its name to TRAIN.DATASETS_NAME within the YAML file under experiments/ostrack.

**Comment by setarekhosravi** [comment]
@Kou-99 
Thank you for your help.
